.coveragerc

```
@@ -0,0 +1,4 @@
+[run]
+omit =
+    */src/genetic_testing/assay_target/primer_design.py
```

```suggestion
    */genetic_testing/assay_target/primer_design.py
```

Try this... looking at the xml, the actual code path might be the above instead.

```
<class name="primer_design.py" filename="/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/genetic_testing/assay_target/primer_design.py" complexity="0" line-rate="0" branch-rate="0">
					<methods/>
					<lines>
						<line number="9" hits="0"/>
						<line number="10" hits="0"/>
						<line number="11" hits="0"/>
						<line number="13" hits="0"/>
```